src/Microsoft.Health.Fhir.SqlServer/Features/Storage/Registry/SearchParameterStatusCollection.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Data;
+using Microsoft.Data.SqlClient.Server;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.Registry
+{
+    public class SearchParameterStatusCollection : List<ResourceSearchParameterStatus>, IEnumerable<SqlDataRecord>
+    {
+        IEnumerator<SqlDataRecord> IEnumerable<SqlDataRecord>.GetEnumerator()
+        {
+            var sqlRow = new SqlDataRecord(
+                new SqlMetaData("Uri", SqlDbType.NVarChar, 128),
+                new SqlMetaData("Status", SqlDbType.NVarChar, 10),
+                new SqlMetaData("IsPartiallySupported", SqlDbType.Bit));
```

Comment:
>IsPartiallySupported [](start = 33, length = 20)

How is `LastUpdated` field get updated?

Reply:
Good question. `LastUpdated` is set in the SQL schema code (see line 60 of `6.diff.sql`).